core/src/nucleus/ribosome/api/mod.rs

```diff
@@ -216,6 +221,13 @@ pub fn call(
             index: usize,
             args: RuntimeArgs,
         ) -> Result<Option<RuntimeValue>, Trap> {
+            // Abort
+            if index == 0 {
+                // println!("message, file, line, column");
```

Comment:
dead code

Reply:
as mentioned, @ddd-mtl and @thedavidmeister I just copied what `NopExternals` does here, and throws 'Unreachable' but that's not actually right. I feel in the dark about rust error handling still. 

@AshantiMutinta maybe you have a few to help me in here on this?

Reply:
absolutely, ping me on mattermost @Connoropolous or send me an email. If you want I can give you a crash course just on how everything works!